src/Sylius/Bundle/ThemeBundle/Command/SyliusThemeDebugCommand.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\ThemeBundle\Command;
+
+use Sylius\Bundle\ThemeBundle\Model\ThemeInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+class SyliusThemeDebugCommand extends ContainerAwareCommand
```

Comment:
I'd say we don't need to do this. It just has to be well documented in the docs that we override standard symfony command. As you can see, I could not figure it out anyway. :P